frontend/src/lib/getStartAndEndTime/index.ts

```diff
@@ -71,6 +71,16 @@ const GetStartAndEndTime = ({
 		};
 	}
 
+	if (type === 'LAST_3_DAYS') {
```

Comment:
We are using the same logic in all `if` conditions, we should consider moving it to a function.